tests/test_component_value_default.py

```diff
@@ -0,0 +1,12 @@
+from formiodata.form import Form
+from test_component import ComponentTestCase
+from formiodata.components import emailComponent
+
+
+class valueDefaultEmailComponent(ComponentTestCase):
```

Comment:
Please rename file to `test_default_value_component`.
So distinguishing the list of test files becomes easier, regarding components and other specific tests.